audformat/core/column.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations  # allow typing without string
```

Comment:
Why do we nedd this? Is it also required for the trick with the circular imports? Don't really understand the comment, though.

Reply:
But here we define the `Column` class so why do we have to write 'Scheme'?

Reply:
So we need it together with

```python
if typing.TYPE_CHECKING:
    # Fix to make mypy work without circular imports,
    # compare
    # https://adamj.eu/tech/2021/05/13/python-type-hints-how-to-fix-circular-imports/
    from audformat.core.scheme import Scheme  # pragma: nocover
```

So maybe I add a remark in the comment that it belongs to this part?

Reply:
Ah ok, I think I get it. Because we import `Scheme` later to avoid circulate imports, we would need to put it in strings. And that we avoid with the future import.

Reply:
Now that I understand how it works, I'd say it's fine :)